### ISSUE TITLE
Support force transfer from issuer

### ISSUE THREAD

**Issue by sschiessl-bcp** [bitshares/bitshares-ui#1892]
This might be only a question ....

Does the UI support to force transfer UIAs if the permission is set?

**Comment by sschiessl-bcp** [comment]
@svk31 does bithsaresjs already support this transfer?

**Comment by svk31** [comment]
Yes it does

**Comment by sschiessl-bcp** [comment]
My suggestion (took 0.5h):
![image](https://user-images.githubusercontent.com/33128181/48715325-0b214c00-ec15-11e8-84f7-78278fab8460.png)

This is a rough scatch, of course style guide best practices and defaults should be used. Remarks:
 - Override as issuer is only visible if the currentAccount is the issuer of the selected asset
 - Override as issuer can be displayed at the top, or under the Quantity field (also if Proposal is checked before)
 - I don't know atm if override supports memo. If not, remove

**Comment by startailcoon** [comment]
Is this a feature requested by the community?

**Comment by sschiessl-bcp** [comment]
No, it's a feature that the backend has and is missing in UI. Maybe and due to the limited user base it could be for asset action site

**Comment by ioBanker** [comment]
Perhaps this way is easier? what do you think

<img width="960" alt="ovveride2" src="https://user-images.githubusercontent.com/37595908/129355998-9172632e-4262-4ddf-a85f-fc6b016d803d.png">
<img width="960" alt="ovveride" src="https://user-images.githubusercontent.com/37595908/129356002-048abce5-6570-4f2a-91df-bc426625bc6f.png">

**Comment by sschiessl-bcp** [comment]
Perfect!
